Civi/ActionSchedule/RecipientBuilder.php

```diff
@@ -314,7 +314,7 @@ protected function buildAddlRepeatPass() {
 
     $addlCheck = \CRM_Utils_SQL_Select::from($query['casAddlCheckFrom'])
       ->select('*')
-      ->merge($query, array('params', 'wheres'))// why only where? why not the joins?
+      ->merge($query, array('params', 'wheres', 'joins'))
```

Comment:
Yes, why not the joins?

Reply:
lol whose prescient code comment was that? I thought @totten when I read it but he would have said 'Dear god why...'

Reply:
From digging through the history, I think it was @totten's comment to himself as he rewrote the code to match the old behavior, not being certain why.

Reply:
I checked with Tim & yeah he was preserving existing code that didn't make sense to him :-)